app/html/websiteAccess.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html style = 'background-color: var(--bg-color); overflow-y: inherit;'>
+	<head>
+		<title>Import settings - The Interceptor</title>
+		<meta charset = 'utf-8'>
+		<link rel = 'icon' type = 'image/x-icon' href = 'favicon.ico'>
+	</head>
+	<body style = 'background-color: var(--bg-color); margin: auto;'>
+		<meta name = 'viewport' content = 'width = device-width, initial-scale = 1' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/bulma.css' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/bulma-divider.css' />
+		<link rel = 'stylesheet' type = 'text/css' href = '../css/interceptor.css' />
+		<script src = '../vendor/webextension-polyfill/browser-polyfill.js'></script>
+		<script type = 'importmap'>
+		{
+			"imports": {
```

Comment:
these are generated with a script in `app/build/vendor.mts`. So you need to add this file to the files list in there to have it updated automatically once new imports are added/removed